product_docs/docs/pgd/5/cli/command_ref/node/get-option.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: pgd node get-option
+navTitle: Get Option
+---
+
+## Synopsis
+
+The `pgd node get-option` command is used to get node-level options in the EDB Postgres Distributed cluster.
+
+## Syntax
+
+```
+pgd node <NODENAME> get-option [OPTIONS] [OPTION]
+```
+
+Where `<NODENAME>` is the name of the node for which you want to get options.
+
+And `[OPTION]` is the name of a specific group option you want to get. If no option is specified, all options are displayed.
+
+The following options are available:
+
+### Node Options
+
+| Option         | Description                        |
+|----------------|------------------------------------|
+| route_priority | Priority assigned to the node.     |
+| route_fence    | Set to fence the node              |
+| route_writes   | Set to allow node to route writes. |
+| route_reads    | Set to allow node to route reads.  |
+| route_dsn      | DSN for connections to this node   |
+
+## Options
+
+No command specific options. See [Global Options](/pgd/latest/cli/command_ref/#global-options).
+
+## Examples
+
+### Get all node options
+
+```shell
+pgd node node-one get-option
```

Comment:
Use the `<>` as `<node-name>` to distinguish it from a value vs a sub-command name